medusa/providers/torrent/torznab/torznab.py

```diff
@@ -310,47 +310,50 @@ def _parse_cap(tag):
 
         self.cap_tv_search = _parse_cap('tv-search')
 
-    def get_categories(self, just_caps=False):
+    def get_capabilities(self, just_caps=False):
         """
         Use the provider url and apikey to get the capabilities.
 
         Makes use of the default newznab caps param. e.a. http://yournewznab/api?t=caps&apikey=skdfiw7823sdkdsfjsfk
         Returns a tuple with (succes or not, array with dicts [{'id': '5070', 'name': 'Anime'},
         {'id': '5080', 'name': 'Documentary'}, {'id': '5020', 'name': 'Foreign'}...etc}], error message)
         """
-        categories = []
+        categories = supported_params = []
+        error_msg = ''
 
         if not self._check_auth():
-            return False, categories, '', 'Provider requires auth and your key is not set'
+            error_msg = 'Provider requires auth and your key is not set'
+            return False, categories, supported_params, error_msg
 
         url_params = {'t': 'caps'}
         if self.needs_auth and self.api_key:
             url_params['apikey'] = self.api_key
 
         response = self.session.get(urljoin(self.url, 'api'), params=url_params)
         if not response or not response.text:
-            error_string = 'Error getting caps xml for [{0}]'.format(self.name)
-            log.warning(error_string)
-            return False, categories, '', error_string
+            error_msg = 'Error getting caps xml for: {0}'.format(self.name)
+            log.warning(error_msg)
+            return False, categories, supported_params, error_msg
```

Comment:
It would be nice if we return a named tuple here.

Reply:
Will add this tomorrow.